src/constructs/lambda/lambda.ts

```diff
@@ -108,6 +139,21 @@ export class GuLambdaFunction extends Function {
       });
     }
 
+    if (props.config) {
+      // For details, see https://github.com/guardian/ssm-to-env.
+
+      const configPrefix = props.config.ssmPrefix ?? `/${scope.stage}/${scope.stack}/${this.app}`;
+      const configLayer = LayerVersion.fromLayerVersionArn(
+        scope,
+        "config-layer",
+        "arn:aws:lambda:eu-west-1:ACCOUNT-INFO-TODO:layer:ssm-to-env-layer:6" // todo how to include account number here?
```

Comment:
Yep.

Reply:
> we don't want the account ID hard coded here?
I wonder whether it could be part of https://github.com/guardian/private-infrastructure-config ... but that would add an unpleasant dependency that would be annoying for consumers.

Reply:
(Resolved via https://github.com/guardian/aws-account-setup/pull/61.)